diff -rU3 64Gram-1.0.88-full.orig/Telegram/lib_base/base/platform/linux/base_system_media_controls_linux.cpp 64Gram-1.0.88-full/Telegram/lib_base/base/platform/linux/base_system_media_controls_linux.cpp
--- 64Gram-1.0.88-full.orig/Telegram/lib_base/base/platform/linux/base_system_media_controls_linux.cpp	2023-07-22 08:18:30.592763000 -0400
+++ 64Gram-1.0.88-full/Telegram/lib_base/base/platform/linux/base_system_media_controls_linux.cpp	2023-08-05 12:44:45.409132095 -0400
@@ -193,7 +193,7 @@
 	player().set_loop_status("None");
 	player().set_rate(1.0);
 	const auto executeCommand = [=](
-			GObject::Object,
+			gi::repository::GObject::Object,
 			Gio::DBusMethodInvocation invocation) {
 		base::Integration::Instance().enterFromEventLoop([&] {
 			_commandRequests.fire_copy(
@@ -233,23 +233,23 @@
 		return true;
 	});
 	player().property_loop_status().signal_notify().connect([=](
-			GObject::Object,
-			GObject::ParamSpec) {
+			gi::repository::GObject::Object,
+			gi::repository::GObject::ParamSpec) {
 		base::Integration::Instance().enterFromEventLoop([&] {
 			_commandRequests.fire_copy(
 				LoopStatusToCommand(player().get_loop_status()));
 		});
 	});
 	player().property_shuffle().signal_notify().connect([=](
-			GObject::Object,
-			GObject::ParamSpec) {
+			gi::repository::GObject::Object,
+			gi::repository::GObject::ParamSpec) {
 		base::Integration::Instance().enterFromEventLoop([&] {
 			_commandRequests.fire_copy(Command::Shuffle);
 		});
 	});
 	player().property_volume().signal_notify().connect([=](
-			GObject::Object,
-			GObject::ParamSpec) {
+			gi::repository::GObject::Object,
+			gi::repository::GObject::ParamSpec) {
 		base::Integration::Instance().enterFromEventLoop([&] {
 			_volumeChangeRequests.fire_copy(player().get_volume());
 		});
