From 17857e06e2141ce4b047ed36fafd91ab441ff5b5 Mon Sep 17 00:00:00 2001
From: crashniels <39709384+crashniels@users.noreply.github.com>
Date: Mon, 19 Jul 2021 17:44:44 +0200
Subject: [PATCH] Add driver info

---
 aura-gpu-hw.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/aura-gpu-hw.c b/aura-gpu-hw.c
index f064b79..b443fb1 100644
--- a/aura-gpu-hw.c
+++ b/aura-gpu-hw.c
@@ -411,6 +411,7 @@ static struct hw_i2c_context *aura_gpu_i2c_create (
     context->atom_context->scratch_size_bytes = sizeof(context->scratch);
     context->adapter.owner = THIS_MODULE;
     context->adapter.class = I2C_CLASS_DDC;
+    context->adapter.dev.parent = &pci_dev->dev;
 
     i2c_set_adapdata(&context->adapter, context);
 
