diff --git a/pci_ids.h b/pci_ids.h
index a94acc8..5a71aa7 100644
--- a/pci_ids.h
+++ b/pci_ids.h
@@ -10,7 +10,10 @@
     NVIDIA devices already have the adapter loaded.
  */
 static const struct pci_device_id pciidlist[] = {
+    {0x1002, 0x67df, 0x1043, 0x04fd, 0, 0, CHIP_POLARIS10},     // RX480 (Strix)
+    {0x1002, 0x67df, 0x1043, 0x04c2, 0, 0, CHIP_POLARIS10},     // RX570 (Strix)
     {0x1002, 0x67df, 0x1043, 0x0517, 0, 0, CHIP_POLARIS10},     // RX580 (Strix)
+    {0x1002, 0x67df, 0x1043, 0x0519, 0, 0, CHIP_POLARIS10},     // RX580 (Strix TOP)
     {0x1002, 0x687F, 0x1043, 0x0555, 0, 0, CHIP_VEGA10},        // Vega 56 (Strix)
     // {0x1002, 0x731f, 0x1043, 0x04e2, 0, 0, CHIP_NAVI10},     // RX5700XT (Strix)
     {0, 0, 0},
